packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/container.tsx

```diff
@@ -45,15 +45,24 @@ const ResetPasswordWithRecoveryKeyVerifiedContainer = ({
 
   const account = useAccount();
   const { uid, sessionToken } = currentAccount() || {};
```

Comment:
On refreshing the page, I happened to see a general application error when trying to retrieve the `email` from location state (a few lines above) - we could instead retrieve that value from the current account in local storage (as we do for `uid` and `sessionToken` here).

Reply:
This is https://github.com/mozilla/fxa/pull/18157 - will update that patch based on the comments you and Lauren left there.